wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantView.swift

```diff
@@ -161,4 +157,29 @@ final class CallParticipantView: BaseCallParticipantView {
             completionBlock(true)
         }
     }
+
+    // MARK: Override Base
+
+    override func updateVideoShouldFill(_ shouldFill: Bool) {
+        if shouldFill, videoView == nil {
+            // [WPB-8954] Setup video only when the video really starts to avoid
+            // calls crashing on the iOS 17 simulator.
+            let videoView = makeVideoView()
+            self.videoView = videoView
+
+            videoContainerView?.setupVideoView(videoView)
```

Comment:
So will this still crash when video is enabled?

Reply:
Unfortunately yes. It just makes it possible to have audio only calls in the simulator again (as on iOS 16 simulators it was working).